modules/persistenceService/src/main/resources/db/migration/V2__ALTER_USER_TABLE_DROP_ADD_NEW_COLUMN.sql

```diff
@@ -12,9 +12,9 @@ CREATE TABLE repositories(
 );
 
 CREATE TABLE subscriptions(
-    id uuid NOT NULL,
+    user_id uuid NOT NULL,
```

Comment:
here I think it'd be better to add a versioned migration:

```sql
alter table subscriptions rename column id to user_id;
```

Reply:
sure will do